sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseDataForm.java

```diff
@@ -309,9 +307,9 @@ protected void addFields() {
 		addField(CaseDataDto.INVESTIGATION_STATUS, NullableOptionGroup.class);
 		addField(CaseDataDto.OUTCOME, NullableOptionGroup.class);
 		addField(CaseDataDto.SEQUELAE, NullableOptionGroup.class);
-		if (isConfiguredServer(CountryHelper.COUNTRY_CODE_GERMANY)) {
-			addField(CaseDataDto.REPORTING_TYPE);
-		}
+
+		addField(CaseDataDto.REPORTING_TYPE);
+		((ComboBox) addField(CaseDataDto.REPORTING_DISTRICT)).addItems(FacadeProvider.getDistrictFacade().getAllActiveAsReference());
```

Comment:
Is this needed for all countries or only for Germany? From the story my understanding is that it should be for Germany only.

Reply:
It will be hidden for other countries because of the @HideForCountrysExcept annotation in CaseDtaDto.

Loading all districts could cause performance issues for other countries but it's not a big query so it should not really affect the performance